R/ebook.R

```diff
@@ -86,7 +86,8 @@ process_markdown = function(
   on.exit(file.remove(intermediate_html), add = TRUE)
   rmarkdown::pandoc_convert(
     input_file, 'html', from, intermediate_html, TRUE,
-    c(pandoc_args, '--section-divs', '--mathjax', '--number-sections')
+    c(pandoc_args, '--section-divs', '--mathjax', '--number-sections',
+      if (rmarkdown::pandoc_available("2.17")) c('--wrap', 'none'))
```

Comment:
```suggestion
    c(pandoc_args2(pandoc_args), '--section-divs', '--mathjax', '--number-sections')
  )
```

This also work it seems. Is it better as we already use `--wrap` elsewhere ?

Reply:
I have made the change